CSAR.lua

```diff
@@ -8,99 +8,29 @@
 --      2 - Disable Aircraft for Pilot when he's shot down -- timeout to reenable pilot for aircraft
 --      3 - Pilot Life Limit - No Aircraft Disabling 
 
+
+
 csar = {}
 
 -- SETTINGS FOR MISSION DESIGNER vvvvvvvvvvvvvvvvvv
-csar.csarUnits = {
```

Comment:
I'd still leave this - as a lot of people name there units still like this - along with your new settings

Reply:
Added it back and tested it..